packages/typespec-python/scripts/eng/utils.ts

```diff
@@ -1,10 +1,28 @@
 /* eslint-disable no-console */
 import { exec } from "child_process";
+import process from "process";
 import { existsSync } from "fs";
 import { dirname, join } from "path";
 import { fileURLToPath } from "url";
 import chalk from "chalk";
 
+// execute the command
+export function executeCommand(command: string, prettyName: string) {
+    exec(command, (error, stdout, stderr) => {
+        if (error) {
+            console.error(chalk.red(`Error executing ${command}(stdout): ${stdout}`));
+            console.error(chalk.red(`Error executing ${command}{stderr}: ${stderr}`));
```

Comment:
Make sure we could get complete info when error occurs like: 
![image](https://github.com/user-attachments/assets/e42bfaed-3da3-4cf4-8c1d-ee1beb2f497f)